src/components/Search.js

```diff
@@ -7,35 +7,10 @@ const Search = props => (
     </div>
     <div className="row">
       <div className="col">
-        <table className="table search-table">
-          <thead>
-            <tr>
-              <th>Customer id</th>
-              <th />
-              <th></th>
-              <th />
-            </tr>
-          </thead>
-          <tbody>
-            <tr>
-              <td>
-                <input
-                  id="customerId"
-                  type="text"
-                  className="form-control"
-                  placeholder="Customer id"
-                  name="customerId"
-                />
-              </td>
-              <td>
-                <button className="btn btn-primary fn-submit-name">Search IDs</button>
```

Comment:
@ameerkabir You need to only move the button out not the whole. 
Change the button on the `Search.js` page to be a separate component